src/Cake.Prca/Orchestrator.cs

```diff
@@ -17,23 +17,23 @@ internal class Orchestrator
         private readonly ICakeLog log;
         private readonly List<ICodeAnalysisProvider> codeAnalysisProviders = new List<ICodeAnalysisProvider>();
         private readonly IPullRequestSystem pullRequestSystem;
-        private readonly ReportIssuesToPullRequestSettings settings;
+        private readonly PrcaSettings settings;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="Orchestrator"/> class.
         /// </summary>
         /// <param name="log">Cake log instance.</param>
         /// <param name="codeAnalysisProviders">List of code analysis issue providers to use.</param>
-        /// <param name="pullRequestSystem">Object for accessing pull request system.</param>
+        /// <param name="pullRequestSystem">Optional object for accessing pull request system.
+        /// <c>null</c> if only issues should be read.</param>
         /// <param name="settings">Settings.</param>
         public Orchestrator(
             ICakeLog log,
             IEnumerable<ICodeAnalysisProvider> codeAnalysisProviders,
             IPullRequestSystem pullRequestSystem,
-            ReportIssuesToPullRequestSettings settings)
+            PrcaSettings settings)
```

Comment:
Do we need this change? It seems like we are using the Orchestrator always with a settings parameter of type `ReportIssuesToPullRequestSettings`.